python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/aaa-accounting.j2

```diff
@@ -12,39 +12,150 @@
+{%                 if method.group is arista.avd.defined and method.method == "group" %}
+{%                     do methods_list.append(method.group) %}
+{%                 endif %}
+{%                 if method.method == "logging" %}
+{%                     set logging_namespace.method_logging = true %}
+{%                 endif %}
+{%             endfor %}
+{%             if methods_list == [] %}
+{%                 do methods_list.append("-") %}
+{%             endif %}
+| Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ ", ".join(methods_list | arista.avd.default(["-"])) }} | {{ logging_namespace.method_logging }} |
+{%         elif aaa_accounting.exec.console.logging is arista.avd.defined or aaa_accounting.exec.console.group is arista.avd.defined %}
+{%             set aaa_accounting_logging = aaa_accounting.exec.console.logging | arista.avd.default("-") %}
+{%             set aaa_accounting_group = aaa_accounting.exec.console.group | arista.avd.default("-") %}
 | Exec - Console | - | {{ aaa_accounting.exec.console.type }} | {{ aaa_accounting_group }} | {{ aaa_accounting_logging }} |
+{%         endif %}
 {%     endif %}
 {%     if aaa_accounting.commands.console is arista.avd.defined %}
 {%         for command_console in aaa_accounting.commands.console %}
-{%             if command_console.commands is arista.avd.defined and command_console.type is arista.avd.defined %}
-{%                 set group = command_console.group | arista.avd.default(" - ") %}
-{%                 set logging = command_console.logging | arista.avd.default("False") %}
+{%             if command_console.type is arista.avd.defined and command_console.commands is arista.avd.defined %}
+{%                 if command_console.type == "none" %}
+| Commands - Console | {{ command_console.commands }} | none | - | - |
+{%                 elif command_console.methods is arista.avd.defined %}
+{%                     set methods_list = [] %}
+{%                     set logging_namespace = namespace(method_logging = false) %}
+{%                     for method in command_console.methods %}
+{%                         if method.group is arista.avd.defined and method.method == "group" %}
+{%                             do methods_list.append(method.group) %}
+{%                         endif %}
+{%                         if method.method == "logging" %}
+{%                             set logging_namespace.method_logging = true %}
+{%                         endif %}
+{%                     endfor %}
+{%                     if methods_list == [] %}
+{%                         do methods_list.append("-") %}
+{%                     endif %}
+| Commands - Console | {{ command_console.commands }} | {{ command_console.type }} | {{ ", ".join(methods_list | arista.avd.default(["-"])) }} | {{ logging_namespace.method_logging }} |
+{%                 elif command_console.group is arista.avd.defined or command_console.logging is arista.avd.defined %}
```

Comment:
ok found it - in the EOS template we are doing this:

```suggestion
{%                 elif command_console.group is arista.avd.defined or command_console.logging is arista.avd.defined(true) %}
```

which explains the difference (we render in doc something not in CLI which can be quite confusing in my opinion). @ClausHolbechArista ok to adjust this in this PR (this could be seen as a "breaking change because the documentation would change without any input change but well ... this is not correct)

Reply:
let's leave it as it is